src/layer/tile/TileLayer.WMS.js

```diff
@@ -97,16 +97,19 @@ export var TileLayerWMS = TileLayer.extend({
 
 	getTileUrl: function (coords) {
 
-		var tileBounds = this._tileCoordsToBounds(coords),
-		    nw = this._crs.project(tileBounds.getNorthWest()),
-		    se = this._crs.project(tileBounds.getSouthEast()),
-
-		    bbox = (this._wmsVersion >= 1.3 && this._crs === EPSG4326 ?
-			    [se.y, nw.x, nw.y, se.x] :
-			    [nw.x, se.y, se.x, nw.y]).join(','),
-
-		    url = TileLayer.prototype.getTileUrl.call(this, coords);
-
+		var tileBounds = this._tileCoordsToNwSe(coords),
+		  nw = this._crs.project(tileBounds[0]),
+		  se = this._crs.project(tileBounds[1]);
+		// in polar projections NS lat/lon box might reversed (S 'above' N) so check and swap if need be
+		if (se.y > nw.y) {

```

Comment:
It will look cleaner if you just turn `nw` and `se` into a [`Bounds`](http://leafletjs.com/reference-1.1.0.html#bounds) instance, that will automatically handle swapping around coordinates. You can then either access them through `min` and `max` properties or its various methods for getting each of the bound's four corners.